packages/commonwealth/client/scripts/views/components/cosmos_proposal_selector/cosmos_proposal_selector.tsx

```diff
@@ -27,25 +26,21 @@ export const ProposalSelector = ({
   onSelect,
   proposalsToSet,
 }: ProposalSelectorProps) => {
-  const [loadingCompleted, setCompletedLoading] = useState(false);
+  const [loadingCompletedProposals, setCompletedProposalsLoading] =
+    useState(false);
   const [loadingActive, setActiveLoading] = useState(false);
-  const [loading, setLoading] = useState(false);
   const [searchTerm, setSearchTerm] = useState('');
   const { activeCosmosProposals } = useGetActiveCosmosProposals({
     app,
-    setIsLoading: setLoading,
+    setIsLoading: setActiveLoading,
     isLoading: loadingActive,
   });
   const { completedCosmosProposals } = useGetCompletedCosmosProposals({
     app,
-    setIsLoading: setCompletedLoading,
-    isLoading: loadingCompleted,
+    setIsLoading: setCompletedProposalsLoading,
+    isLoading: loadingCompletedProposals,
   });
 
-  useEffect(() => {
-    setLoading(!(activeCosmosProposals.length > 0));
```

Comment:
This was causing infinite bobber if no active proposals. I think my changes here are still what we are looking for.

Reply:
Confirmed fixed on chihuahua.